scripts/west_commands/runners/nrfjprog.py

```diff
@@ -81,6 +95,9 @@ def do_exec_op(self, op, force=False):
             if _op.get('verify'):
                 # In the future there might be multiple verify modes
                 cmd.append('--verify')
+            if self.qspi_ini:
```

Comment:
@carlescufi wouldn't this add --qspiint option to any call to nrfjprog in a row within one `west flash` proceddure?

Reply:
No, because this is only in the `elif op_type == 'program':` section.